src/sites/link/linkdrop.net.js

```diff
@@ -269,6 +265,13 @@
         attributes: true,
       });
 
+      await _.wait(1000);
+      const click = f.clientWidth === 0 || f.childNodes.length === 0;
+      if (click && !b.disabled) {
+        _.info('clicking submit button, because recaptcha was empty');
+        b.click();
+      }
```

Comment:
Don't we need to constantly check whether the button ready or not?

Reply:
This change is for sites that have an invisible captcha or an empty captcha container.
(e.g. http://wicr.me/0KLTtqmJm ; test it in private mode, too)

I doubt the captcha container will clear itself after some time.